zadani/soft/index.html

```diff
@@ -1,72 +1,86 @@
 <!DOCTYPE html>
 <html>
-    <head>
-        <meta charset="utf-8">
-        <meta name="viewport" content="width=device-width, initial-scale=1">
-        <title>Cestování po Islandu</title>
-        <link href="https://fonts.googleapis.com/css?family=Merriweather:400,400i|Raleway:400,700" rel="stylesheet">
-        <link rel="stylesheet" href="style.css">
-    </head>
-    <body>
 
+<head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <title>Cestování po Islandu</title>
+    <link href="https://fonts.googleapis.com/css?family=Merriweather:400,400i|Raleway:400,700" rel="stylesheet">
+    <link rel="stylesheet" href="style.css">
+</head>
 
-        <!-- hlavička -->
+<body>
 
-        <h1 class="titulek">Island</h1>
 
-        <p class="perex">Když se řekne Island, spoustě lidem se jako první vybaví sníh a led. Sama jsem tam strávila téměř rok života a zem pokrytou sněhem a ledem jsem zažila jen jediný den. Jednou věcí si ale jistá jsem: že návštěva ostrova pod polárním kruhem je pro každého tak silným zážitkem, že se sem bude chtít vrátit.</p>
+    <!-- hlavička -->
+    <div class=cela_stranka>
 
-        <p class="autor">Jana Zcela Vymyšlená</p>
+        <div class="hlavicka">
+            <h1 class="titulek">Island</h1>
 
+            <p class="perex">Když se řekne Island, spoustě lidem se jako první vybaví sníh a led. Sama jsem tam strávila téměř rok života a zem pokrytou sněhem a ledem jsem zažila jen jediný den. Jednou věcí si ale jistá jsem: že návštěva ostrova pod polárním kruhem je pro každého tak silným zážitkem, že se sem bude chtít vrátit.</p>
 
+            <p class="autor">Jana Zcela Vymyšlená</p>
 
+        </div>
 
-        <!-- menu -->
 
-        <ul class="menu">
-            <li><a href="#">Úvod</a></li>
-            <li><a href="#">Proč jet na Island</a></li>
-            <li><a href="#">Co musíte vidět</a></li>
-            <li><a href="#">Fotogalerie</a></li>
+</body>
+<!-- menu -->
+
+<div class="obsah">
+
+    <div class="menu">
+        <ul class="menu_seznam">
```

Comment:
Tady by možná mohla být struktura jednodušší - v menu máš jen 1 prvek (ul), takže by to podle mě šlo i bez obalení 🙂